source/configuration/read.txt

```diff
@@ -196,8 +186,9 @@ Partitioner Configuration
      - Description
 
    * - ``partitionKey``
```

Comment:
All partitioner configuration has the following prefix: `partitioner.options.` so it should be `partitioner.options.partitionKey`